android/src/main/java/com/reactnativecommunity/imageeditor/ImageEditorModuleImpl.kt

```diff
@@ -265,7 +267,14 @@ class ImageEditorModuleImpl(private val reactContext: ReactApplicationContext) {
     }
 
     private fun openBitmapInputStream(uri: String): InputStream? {
-        return if (isLocalUri(uri)) {
+        return if (uri.startsWith("data:")) {
+            val src = uri.substring(uri.indexOf(",") + 1)
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                ByteArrayInputStream(Base64.getMimeDecoder().decode(src))
+            } else {
+                ByteArrayInputStream(android.util.Base64.decode(src, android.util.Base64.DEFAULT))
```

Comment:
Nit: how about giving an alias for that android Base64, so that it doesn't conflict with Java Base64?

`import android.util.Base64 as AndroidUtilBase64`

Reply:
good idea !